crates/matrix-sdk/src/client/mod.rs

```diff
@@ -1928,6 +1928,26 @@ impl Client {
         self.send(request, None).await
     }
 
+    /// Change the display name of a device owned by the current user.
+    ///
+    /// Returns a `update_device::Response` which specifies the result
+    /// of the operation.
+    ///
+    /// # Arguments
+    ///
+    /// * `device` - The `Device` to change the display name of.
+    /// * `display_name` - The new display name to set.
+    pub async fn rename_device(
+        &self,
+        device: &OwnedDeviceId,
```

Comment:
Same as above, let's call the parameter `device_id`.

Reply:
Also, this should be `&DeviceId`. `OwnedDeviceId` derefs to `DeviceId`.